src/inputCarName.js

```diff
@@ -0,0 +1,39 @@
+import { showView } from "./initialization.js";
+
+function inputCarName() {
+  // car name을 입력받고 validation 체크
+  const carsNameInput = document.getElementById("car-names-input");
+  const carNameSubmitBtn = document.getElementById("car-names-submit");
+  
+  
+  carNameSubmitBtn.addEventListener('click', handleSubmit);
+
+  function handleSubmit() {
+    validationCarName();
+  };
+
+
+  function validationCarName() {
+    // input tag 값에 대한 유효성 체크 
+    let inputStr = carsNameInput.value;
+    let inputSplit = inputStr.split(',');
+    let validationFlagValue = true;
+
+    // 길이 체크
+    for(let i in inputSplit) {
+      if(inputSplit[i].length > 5) {
+        alert('자동차 이름의 길이를 5자 이하로 해주세요.');
+        validationFlagValue = false;
+        break;
+      }
+    }
```

Comment:
[https://github.com/airbnb/javascript#iterators-and-generators](url) 
여기 보시면 **iterator** 관련해서 권장하는(?) 사용법이 있습니다! (11.1에 있어요.)

물론 충분히 이해가 가는 코드이고, 좋은 iterator 인데, **forEach** 구문을 쓰면 더 좋다고 명시가 되어 있더라구요!
그와 동시에 화살표 함수를 쓰는 것이 이유는 아직은 잘 모르겠는데 많이들 추천합니다. 

따라서 해당 구문을 고쳐보면
```suggestion
    inputSplit.forEach( (carName) => {
       if(carName.length > 5) {
          alert("자동차 이름의 길이를 5자 이하로 해주세요.");
          validationFlagValue = false;
       }
    }) 
```

이런식으로 수정할 수 있을것 같아요! 다만.. 감히 팁을 드리자면! **forEach** 구문은 중간에 break하기가
어렵다는 단점이 있습니다!